tools/pdf2txt.py

```diff
@@ -161,15 +171,52 @@ def convert_from_pdf(filenames: List[str],
                                           check_extractable=True):
                 page.rotate = (page.rotate + params.rotation) % 360
                 interpreter.process_page(page)
+
     device.close()
     outfp.close()
 
 
+def split_by_chapters(file_name: str) -> List[str]:
+    if not file_name.endswith('.txt'):
+        raise Exception('File type must be Text File')
+    with open(file_name, 'r', encoding='utf-8') as f:
+        lines = f.readlines()
+
+    chapter = 0
+
+    chapters = [""]
+
+    for line_nr in range(0, len(lines) - 1):
+
+        curr_line = lines[line_nr]
+        next_line = lines[line_nr + 1]
+
+        if 'Chapter' in curr_line and next_line == '\n':
+            is_number = re.compile(r'\d+$')  # Find number and only number
+            chapter_number = curr_line.split('Chapter')[1].strip()
+
+            if is_number.match(chapter_number):
+                chapter += 1
+                chapters.append('')
+        chapters[chapter] += curr_line
+
+    return chapters
+
+
+def write_chapters_to_files(chapters: List[str], path='') -> None:
+    index = 0
+    for chapter in chapters:
+        chapter_name = 'introduction.txt' if index == 0 else 'chapter' + str(index) + '.txt'
+        with open(path + chapter_name, 'w', encoding='utf-8') as f:
+            f.write(chapter)
+        index += 1
+
+
 def main(argv):
     try:
         options, filenames = getopt.getopt(
             argv[1:],
-            'dP:o:t:O:c:s:R:Y:p:m:SCnAVM:W:L:F:'
+            'dP:o:t:O:c:s:R:Y:p:m:SCnAVM:W:L:F:h:'
```

Comment:
Might be a good idea to update the readme since you added a flag

Reply:
Good point. I have added it.